packages/govuk-frontend/src/govuk/init.mjs

```diff
@@ -54,6 +54,20 @@ function initAll(config) {
   })
 }
 
+/**
+ * @template {CompatibleClass} T
+ * @callback onErrorCallback
+ * @param {Error} error - Thrown error
+ * @param {ErrorContext<T>} context - Object containing the element, component class and configuration
+ */
+
+/**
+ * @template {CompatibleClass} T
+ * @typedef {object} createAllOptions
+ * @property {Element | Document} [scope] - scope of the document to search within
+ * @property {onErrorCallback<T>} [onError] - callback function if error throw by component on init
+ */
```

Comment:
**question** We've tended to put our types at the bottom of the files across the project, is there something forcing these to be early on? If not, would be good to align with the rest of the files for consistency.

**suggestion** We've generally been PascalCasing the types rather than camelCasing them.